tubearchivist/home/src/download/yt_dlp_handler.py

```diff
@@ -312,6 +312,8 @@ def _build_obs_user(self):
         """build user customized options"""
         if self.config["downloads"]["format"]:
             self.obs["format"] = self.config["downloads"]["format"]
+        if self.config["downloads"]["format_sort"]:
+            self.obs["format_sort"] = self.config["downloads"]["format_sort"].split(",")
```

Comment:
I think yt-dlp is picky here, I think they expect no spaces after the commas. But we humans have that kind of ingrained. Would be a nice quality of life if we can handle both here, something like:
```python
format_sort_list = [i.strip() for i in format_sort.split(",")]
```

Reply:
Good point, done. I hope the style is OK.